src/osd/scrubber/osd_scrub.h

```diff
@@ -98,29 +98,22 @@ class OsdScrub {
    *  locking: not using the jobs_lock
    */
   void update_job(
-      Scrub::ScrubJobRef sjob,
+      Scrub::ScrubJob& sjob,
```

Comment:
This seems to have neither implementation nor callers.

Reply:
Indeed...
Forgot to rebase with the '.h' changes when removing from the '.cc'.
Fixed.